rules/naming/src/Naming/ExpectedNameResolver.php

```diff
@@ -148,6 +150,26 @@ public function resolveForAssignNew(Assign $assign): ?string
         return $this->propertyNaming->getExpectedNameFromType($fullyQualifiedObjectType);
     }
 
+    /**
+     * @param MethodCall $node
+     */
+    public function resolveForMethodCall(Node $node): ?string
```

Comment:
```suggestion
    public function resolveForMethodCall(MethodCall $methodCall): ?string
```

If it's really a `MethodCall`, it can be inlined... maybe next Rector rule? :D

Reply:
My bad here, it also could be StaticCall